modules/custom/cu_core/cu_core.install

```diff
@@ -679,6 +679,15 @@ function cu_core_update_7019() {
 }
 
 /**
+ * Core Update 7020 for Express 2.4.0.
+ *
+ * Revert cu_block_theme feature.
+ */
+function cu_core_update_7020() {
+  cu_core_revert_features(array('cu_block_theme'));
```

Comment:
should be `cu_core_revert_features(array('cu_block_theme' => 'cu_block_theme'));` to work